Probe/appHandler.go

```diff
@@ -0,0 +1,95 @@
+/*
+ *  Copyright 2020 Google LLC
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package main
+
+import (
+	"os/exec"
+	"strings"
+	"time"
+)
+
+func findDevice() (string, error) {
+	// Assumes the first AVD on the list is the one to be used
+	out, err := exec.Command("emulator", "-list-avds").Output()
+	if err != nil {
+		return "", err
+	}
+	dev := strings.Split(string(out), "\n")
+	return dev[0], nil
+}
+
+func startEmulator() error {
+	dev, err := findDevice()
+	if err != nil {
+		return err
+	}
+	err = exec.Command("emulator", "-avd", dev).Start()
+	time.Sleep(10 * time.Second)
```

Comment:
Why sleep 10 seconds?

Reply:
Maybe make this a flag as well. https://golang.org/pkg/flag/

Reply:
Maybe grab the comamnd and Wait() for it to complete.

Reply:
addressed in 765a7ee